File: js/Sort/insertion.sort.js
const arr = [10,205,4,304,548,39]


console.log("Original array is: ",arr)

for(let i = 1; i<arr.length; i++){
    let key = arr[i];
    let j = i - 1;

    while(j >= 0 && arr[j] > key){
        arr[j + 1] = arr[j];
        j--;
    }

    arr[j + 1] = key
}


console.log("Sorted array is: ",arr)

File: python/sort/insertion.sort.py
arr = [20,40,30,100,5023,409,48,4348,394]

size = len(arr)

print("Original array is: ",arr)

for i in range(1,size):
    key = arr[i]
    j = i - 1
    
    while j >= 0 and key < arr[j]:
        arr[j + 1] = arr[j]
        j -= 1
    arr[j + 1] = key
    
print("Sorted array is: ",arr)